packages/recipe/src/components/EzLabel/Documentation/Stories/Default.stories.tsx

```diff
@@ -11,5 +61,21 @@ export default meta;
 type Story = StoryObj<typeof EzLabel>;
 
 export const Default: Story = {
-  render: () => <div>Coming soon...</div>,
+  args: {},
+  parameters: {
+    playroom: {
+      code: dedent`
+        <div>
+          <EzLabel use="primary" htmlFor="primary-input">Label</EzLabel>
+          <input id="primary-input" placeholder="Some text" />
+        </div>
+      `,
+    },
+  },
+  render: (args: Props) => (
```

Comment:
I've been renaming these where needed to make the exports more explicit.

```suggestion
  render: (args: EzLabelProps) => (
```